docs/source/run.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Run the MCP server
+---
+
+## Run the MCP server binary
+
+```sh showLineNumbers=false
+apollo-mcp-server [OPTIONS] <PATH/TO/CONFIG/FILE>
+```
+
+You can configure the Apollo MCP server using a [YAML configuration file](/apollo-mcp-server/config-file).
+
+If the file is not provided, environment variables for your Apollo graph credentials (`APOLLO_GRAPH_REF` and `APOLLO_KEY`) are required for the server to run.
+
+### CLI options
+
+| Option          | Description               |
+| :-------------- | :------------------------ |
+| `-h, --help`    | Print help information    |
+| `-V, --version` | Print version information |
+
+## Run the MCP server with `rover dev`
+
+You can use the [`rover dev`](/rover/commands/dev) command of Rover CLI `v0.35` or later to run an Apollo MCP Server instance for local development alongside your local graph. Use the `--mcp` flag to start an MCP server and provide an optional configuration file.
+
+```sh
+rover dev --mcp <PATH/TO/CONFIG> 
+```
+
+## Run the MCP server with Docker
+
+1. Modify the MCP server configuration in `graphql/TheSpaceDevs/config.yaml`. Update the paths to point to the `/data` directory, as the Docker container expects:
+    ```yaml title="graphql/TheSpaceDevs/config.yaml" {7, 10}
+    endpoint: https://thespacedevs-production.up.railway.app/
```

Comment:
I recommend updating this setting as well to point to the user's GraphQL API URL or Apollo Router URL. Since instructions involve updating the schema file, this endpoint won't work, so adjusting the URL will help prevent issues.